parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -98,7 +98,7 @@ installStandaloneContainerd() {
     
     #if there is no containerd_version input from RP, use hardcoded version
     if [[ -z ${CONTAINERD_VERSION} ]]; then
-        CONTAINERD_VERSION="1.4.8"
+        CONTAINERD_VERSION="1.5.5"
```

Comment:
we need to gate this on the actual kubernetes version, I think we can use bash + KUBERNETES_VERSION or else use Go templates.

Reply:
Ok. So if version < 1.22, we use containerd 1.4.8 and if it's >= to it then use containerd 1.5.5?
Is the latest containerd not backwards compatible with the earlier k8s versions?

Reply:
it's not a compatibility issue, I think that's just how PM want to ship the change (containerd 1.5 + k8s 1.22 as a coupled pair). clusters with older k8s versions should keep what they have (barring hotfixes)